tools/src/test/python/dlpx/virtualization/_internal/test_cli.py

```diff
@@ -385,6 +418,43 @@ def test_with_dev_fail(mock_build, plugin_config_file, artifact_file,
         assert result.exit_code == 2
         assert not mock_build.called, 'build should not have been called'
 
+    @staticmethod
+    @pytest.mark.parametrize('plugin_config_file',
+                             [''])
+    def test_empty_config_file(plugin_config_file):
+        runner = click_testing.CliRunner()
+        result = runner.invoke(cli.delphix_sdk,
+                               ['build', '-c', plugin_config_file])
+
+        assert result.exit_code == 2
+        assert result.output == (u"Usage: delphix-sdk build [OPTIONS]"
+                                 u"\nTry 'delphix-sdk build -h' for help."
+                                 u"\n"
+                                 u"\nError: Invalid value for '-c' /"
+                                 u" '--plugin-config': File"
+                                 u" ''"
+                                 u" is a directory."
+                                 u"\n")
```

Comment:
This seems like it might be kinda brittle.  In future, maybe the exact error message could change in spacing, linebreaks, punctuation, etc.  And if so, I think this test would break, and then someone would have to come and change it. 

Would it be better to just make sure the string contains, say "Invalid value" and "is a directory" somewhere inside, but doesn't test the whole string for equality?